README.md

```diff
@@ -10,7 +10,7 @@ To get started with this project, you need to have Flutter installed on your mac
 
 1. Clone the application:
     ```bash
-    git clonegit@github.com:droidconKE/flutterconKEApp.git
+    git clone https://github.com/droidconKE/flutterconKEApp.git
```

Comment:
The `git@` option is preferred to allow pushing via SSH. 

Update this to:
`git clone git@github.com:droidconKE/flutterconKEApp.git` instead.